packages/advisor-components/src/RuleDetails/RuleDetails.tsx

```diff
@@ -0,0 +1,247 @@
+              </a>
+            </StackItem>
+          )}
+          {topics && rule.tags && topicLinks(rule as RuleContentRhel, topics).length > 0 && (
+            <StackItem>
+              <strong>{intl.formatMessage(messages.topicRelatedToRule)}</strong>
+              <br />
+              {barDividedList(topicLinks(rule as RuleContentRhel, topics))}
+            </StackItem>
+          )}
+          {isDetailsPage && onVoteClick && <RuleRating {...{ ruleId: rule.rule_id, ruleRating: rule.rating, onVoteClick }} />}
+          {!isDetailsPage && onViewAffectedClick && renderViewAffected()}
+        </Stack>
+      </SplitItem>
+      <SplitItem>
+        <Stack>
+          {children && <StackItem>{children}</StackItem>}
+          <StackItem>
+            <Stack className="ins-c-rule-details__stack">
+              <StackItem>
+                <strong>{intl.formatMessage(messages.totalRisk)}</strong>
+              </StackItem>
+              <StackItem className="pf-u-display-inline-flex alignCenterOverride pf-u-pb-sm pf-u-pt-sm">
+                <span className="ins-c-rule-details__stackitem">
+                  <span>
+                    <InsightsLabel value={rule.total_risk} rest={{}} />
+                  </span>
+                  <Stack hasGutter className="description-stack-override">
+                    <StackItem>
+                      <TextContent>
+                        <Text component={TextVariants.p}>
+                          {intl.formatMessage(messages.rulesDetailsTotalRiskBody, {
+                            risk: intl.formatMessage(messages[TOTAL_RISK_LABEL_KEY[rule.total_risk]] || messages.undefined).toLowerCase(),
+                            strong: (str) => <strong>{str}</strong>,
+                          })}
+                        </Text>
+                      </TextContent>
+                    </StackItem>
+                    <Stack>
+                      <StackItem>
+                        <SeverityLine
+                          className="severity-line"
+                          title={intl.formatMessage(messages.likelihoodLevel, {
+                            level: intl.formatMessage(messages[LIKELIHOOD_LABEL_KEY[rule.likelihood as Likelihood]]),
+                          })}
+                          value={rule.likelihood}
+                          tooltipMessage={intl.formatMessage(messages.likelihoodDescription, {
+                            level: intl.formatMessage(messages[LIKELIHOOD_LABEL_KEY[rule.likelihood as Likelihood]]).toLowerCase(),
+                          })}
+                        />
+                      </StackItem>
+                      <StackItem>{renderImpactSeverityLine()}</StackItem>
+                    </Stack>
+                  </Stack>
+                </span>
+              </StackItem>
+              {resolutionRisk && resolutionRiskDesc && (
+                <React.Fragment>
+                  <hr></hr>
```

Comment:
Hmmm, can this be achieved with styling?

Reply:
What do you mean exactly? This is already achieved by defining styles for the line in a SCSS file

Reply:
Well `hr` element just draws a line, it would be better if we could write the line in CSS instead of by element. The thing is HTML should be used to display the content and not to draw some lines, breaks and such, for this CSS should be used. I would understand it if there would have been a big wall of text afterall the w3c definition for `hr` is `thematic changes in the content`. Is this thematic change in the content?

Reply:
So you propose to replace it with smth like `span` and apply the same styles that were already made here https://github.com/RedHatInsights/frontend-components/pull/1493/files/f20483172ca3844413ddf2c58e6ac926127931a4#diff-89292165f3efa1dbd55471691761cd20dab9a9bfb5fab6eeb832e63e5d5a71caR3 ?

To your last question: I am rewriting the legacy code and design implementation. The line divides a group with two severity lines for issue likelihood and impact number and a group with the only severity line showing the resolution risk. Can't say how much these values are thematically different, but this was reflected in the old RHEL Advisor mock-ups. @doruskova probably can provide some input here :)

Reply:
I don't have anything to add here.

Reply:
@gkarat yeah, span with table borders would be awesome! I understand it's a bit legacy code, but while we're in it let's polish it a bit.

Reply:
Thank you, revisitied it in https://github.com/RedHatInsights/frontend-components/pull/1493/commits/3679ee927d574ef32c0f188682f8c7ab2819dfec